content/faq.md

```diff
@@ -73,24 +73,39 @@ Those providers should only be offered to users if there is a certain reason for
 Providers in this category cannot, if no registration is supported, or should not, because of bad properties, be used for registrations.
 Those providers should only be used for autocomplete (e.g., while adding a contact).
 
+## Where do we have the providers' properties from?
+
+For each property, there must be a corresponding source.
+In most cases, that is a statement on a provider's website or the result page of a rating service.
+The source of a property can also be the property itself.
+That is the case when the property is a web page (e.g., a web registration page).
+
+All sources are stored in the {{< external-link text="providers file" url="https://invent.kde.org/melvo/xmpp-providers/-/blob/master/providers.json" >}}.
+That file is the data source of all provider properties.
+This website's provider listings and details are based on it.
+
 {{< spacer size="middle" >}}
 
 ## Glossary
 
 Unfamiliar terms are explained here.
 
-#### Bus Factor
+### Unknown
+
+A property is called *unknown* if there is no [source](#where-do-we-have-the-providers-properties-from) needed for transparency and maintainability of the data.
```

Comment:
[...] no source *available* for transparency and [...]

Or am I misunderstanding this?

Reply:
Thanks. I improved the wording.